packages/components/modals/ModalBase.tsx

```diff
@@ -39,6 +39,7 @@ type ModalBaseProps = {
   width?: number;
   visible?: boolean;
   Header?: ComponentType;
+  hideHeader?: boolean;
```

Comment:
When you change a base like that, so add a specific props, if we want to respect an atomic design pattern (that what we want) you have to create a specific component on top of that (i'll write pseudo code to show you):
```js
ModalWithoutHeader () => {
    <ModalBase hideHeader ... />
}
```

And on your screens you have to call the specific component so `ModalWithoutHeader` 👍

Reply:
Specific component added